.github/workflows/release.yml

```
@@ -10,10 +10,43 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Set up JDK 1.11
-        uses: actions/setup-java@v1
+      - name: apt-get update
+        run: sudo apt-get update
+      - name: Install gnupg
+        run: sudo apt update && sudo apt install -y gnupg
```

just a nit, but you're running apt-update already in the previous step
```suggestion
        run: sudo apt install -y gnupg
```